src/components/modules/Search.js

```diff
@@ -36,11 +36,18 @@ const SearchLogic = () => {
 
     return (
         <div className="SearchLogic">
-            <form onSubmit={getSearch} className="Search-form">
-                <input className="search-bar" type="text" value={search} onChange={updateSearch} />
-                <button className="search-button"
-                    type="submit">Search</button>
-            </form>
+            <FormWrapper>
+                <form onSubmit={getSearch} className="SearchForm">
```

Comment:
You don't seem to be using these CSS classes. Usually they are not necessary when using styled components.